doc.html

```diff
@@ -314,7 +328,7 @@ <h2>7.1	Irrigation Application File Format</h2>
 |     +-----------------+---------------------------------+
 |     | geometry        | "GeoPackage Geometry"           |
 |     +-----------------+---------------------------------+
-|     | amount          | "amount in m<sup>3</sup> (REAL)"           |
+|     | amount          | "amount in mm (INTEGER)"        |
```

Comment:
In API, amounts should always be m<sup>3</sup>. In user interface, this can be converted to mm for farmer convenience